packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -163,11 +167,24 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
       from: fromAddress,
     },
   ])
+  const [initializerAbiRG, transferImplOwnershipAbiRG] = checkAndReturnInitializationAbi(
```

Comment:
this looks like it returns more than just initialization ABI?
maybe break into 2 functions?